src/Database/LSMTree/Internal/MergeSchedule.hs

```diff
@@ -114,6 +94,7 @@ data MergeTrace =
   | TraceCompletedMerge
       NumEntries -- ^ Size of output run
       RunNumber
```

Comment:
I just noticed we're only tracing this message in the `OneShot` case. Maybe we should trace it in the `Incremental` case as well?

Reply:
I also noticed it. It's part of a bigger question about tracing and would require a more invasive change, like moving at least part of the `MergeTrace` type into the `MergingRun` module. I can add a TODO at least.